aws/rust-runtime/aws-inlineable/Cargo.toml

```diff
@@ -34,7 +37,7 @@ http = "0.2.9"
 http-body = "0.4.5"
 md-5 = "0.10.1"
 ring = "0.16"
-tokio = { version = "1.23.1", features = ["full"] }
+tokio = { version = "1.23.1" }
```

Comment:
I think you need `default-features = false` here. I'm not sure that the default-features are wasm-compat, and I wouldn't count on it.

Reply:
this cargo.toml isn't actually used in anything